eventstore/dynamodb/eventstore.go

```diff
@@ -65,14 +69,26 @@ func (c *EventStoreConfig) provideDefaults() {
 	if c.Region == "" {
 		c.Region = "us-east-1"
 	}
+	if c.Endpoint == "" {
+		c.Endpoint = "http://localhost:8000"
+	}
+	if c.Username == "" {
+		c.Username = "doesnotmatter"
+	}
+	if c.Password == "" {
+		c.Password = "doesnotmatter"
+	}
 }
 
 // NewEventStore creates a new EventStore.
 func NewEventStore(config *EventStoreConfig) (*EventStore, error) {
 	config.provideDefaults()
 
+	creds := credentials.NewStaticCredentials(config.Username, config.Password, "")
```

Comment:
You can fund the full documentation on the recommended auth config here: http://docs.aws.amazon.com/sdk-for-go/api/